app/uk/gov/hmrc/soletraderidentificationfrontend/forms/PersonalDetailsForm.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.soletraderidentificationfrontend.forms
+
+import play.api.data.Form
+import play.api.data.Forms._
+
+object PersonalDetailsForm {
```

Comment:
Doesn't look like this form is being used at the moment, as we're doing the validation and data storage in another story, it might be easier to do the whole form later on?

May be able to delete this file for now.